webapp/src/components/loading/loading.scss

```diff
@@ -0,0 +1,3 @@
+.loading-text {
+    margin: 20px 0;
+}
```

Comment:
This could have an impact on the core web application or any other plugin if they happen to use this css class. We should either have class name include `jira` or just put stuff in `style`